stablehlo/dialect/StablehloOps.td

```diff
@@ -269,7 +269,8 @@ def StableHLO_ConvertOp : StableHLO_UnaryElementwiseOp<"convert",
 }
 
 def StableHLO_ClzOp: StableHLO_UnaryElementwiseOp<"count_leading_zeros",
-    [Pure, HLO_CompatibleOperandsAndResultType], HLO_IntTensor> {
+    [Pure, HLO_CompatibleOperandsAndResultType /*count_leading_zeros_c1*/],
+     HLO_IntTensor /*count_leading_zeros_i1*/> { /*count_leading_zeros_c1*/
```

Comment:
This is just i1 right? Not sure why c1 is included.

Reply:
`count_leading_zeros_c1` is included for the `StableHLO_UnaryElementwiseOp` as the `SameOperandsAndResultShape` trait in that class covers the shape portion of (C1).

Reply:
I see. Please ignore my comment in that case.